homeassistant/components/onboarding/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "onboarding",
   "name": "Home Assistant Onboarding",
   "codeowners": ["@home-assistant/core"],
-  "dependencies": ["auth", "http", "person"],
+  "dependencies": ["auth", "http"],
```

Comment:
Is it really safe to remove this, or do we now introduce a race?

Reply:
moved to https://github.com/home-assistant/core/pull/142585, details there